### PR TITLE
Skeleton

### PR DESCRIPTION
**Pull request by afonsobspinto**
- Add HNN-UI skeleton
    - [x] Setup
    - [x] Docker
    - [x] Travis

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+RUN /bin/bash -c "source activate snakes && python setup.py install"
+
+ARG INCUBATOR_VER=unknown
+RUN /bin/bash -c "INCUBATOR_VER=${INCUBATOR_VER} source activate snakes && pip install netpyne_ui"
+RUN /bin/bash -c "source activate snakes && jupyter nbextension enable --py jupyter_geppetto"
+RUN /bin/bash -c "source activate snakes &&  jupyter serverextension enable --py jupyter_geppetto"
+RUN /bin/bash -c "source activate snakes && jupyter nbextension enable --py widgetsnbextension"
+
+WORKDIR /home/jovyan
+RUN git clone --branch skeleton https://github.com/MetaCell/HNN-UI.git
```
Comment (afonsobspinto) [inline Dockerfile]: @afonsobspinto must change this to development
```suggestion
RUN git clone --branch development https://github.com/MetaCell/HNN-UI.git
```

File: Dockerfile_dev [hunk 2]
```diff
@@ -0,0 +1,61 @@
+FROM jupyter/base-notebook:eb70bcf1a292
+USER root
+
+ARG hnnuiBranch=skeleton
```
Comment (afonsobspinto) [inline Dockerfile_dev:4]: @afonsobspinto change this to development
```suggestion
ARG hnnuiBranch=development
```

File: utilities/install.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+subprocess.call(['pip', 'install', '-e', '.'], cwd='./netpyne/')
+
+clone('https://github.com/openworm/org.geppetto.frontend.jupyter.git', 'org.geppetto.frontend.jupyter', 'refactor-sync')
+subprocess.call(['npm', 'install'], cwd='./org.geppetto.frontend.jupyter/js')
+subprocess.call(['npm', 'run', 'build-dev'], cwd='./org.geppetto.frontend.jupyter/js')
+
+# subprocess.call(['git', 'submodule', 'update', '--init'], cwd='./')
+clone('https://github.com/openworm/org.geppetto.frontend.git', 'geppetto', 'refactor-sync', 'hnn_ui/', False, 'geppetto')
+# checkout('geppetto', 'development','org.geppetto.frontend.jupyter/src/jupyter_geppetto/')
+clone('https://github.com/MetaCell/geppetto-hnn.git', 'geppetto-hnn', 'skeleton',
```
Comment (afonsobspinto) [inline utilities/install.py:65]: @afonsobspinto Change skeleton to development after https://github.com/MetaCell/geppetto-hnn/pull/54
```suggestion
clone('https://github.com/MetaCell/geppetto-hnn.git', 'geppetto-hnn', 'development',
```

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,57 @@
+FROM jupyter/base-notebook:eb70bcf1a292
```
Comment (tarelli) [inline Dockerfile]: We can use this other jupyter-neuron docker now as a starting point (what you asked at the meeting that is now available) https://github.com/MetaCell/NetPyNE-UI/blob/refactor-dockerimage/Dockerfile

File: Dockerfile_dev [hunk 5]
```diff
@@ -0,0 +1,61 @@
+FROM jupyter/base-notebook:eb70bcf1a292
```
Comment (tarelli) [inline Dockerfile_dev]: Ditto

File: utilities/install.py (hunk 3 above)
Comment (tarelli) [reply utilities/install.py:65]: 👍

File: Dockerfile (hunk 1 above)
Comment (tarelli) [reply Dockerfile]: 👍

File: Dockerfile_dev (hunk 2 above)
Comment (tarelli) [reply Dockerfile_dev:4]: 👍

**Review by tarelli — CHANGES_REQUESTED** [review]
Looks great, small changes for the docker!

File: Dockerfile (hunk 4 above)
Comment (afonsobspinto) [reply Dockerfile]: Done + I added a comment to make it obvious to everyone, that this Dockerfile is just a template and is not intended to be functional at the moment.

File: Dockerfile_dev (hunk 5 above)
Comment (afonsobspinto) [reply Dockerfile_dev]: Done but as great professional :sweat_smile: I tested this dockerfile locally and although it builds everything perfectly when I run it I got the error I told you some meetings ago: `ReferenceError: React is not defined` which locally I solved by adding 
```
 new webpack.ProvidePlugin({
            "React": "react",
        }),
```
to the webpack.config.js, but for the given scenario I don't know exactly how to proceed.

**Pull request merged**